packages/govuk-frontend/src/govuk/components/skip-link/skip-link.mjs

```diff
@@ -31,28 +32,43 @@ export class SkipLink extends GOVUKFrontendComponent {
     this.$module = $module
 
     // Check for linked element
-    const $linkedElement = this.getLinkedElement()
-    if (!$linkedElement) {
-      return
+    try {
+      const $linkedElement = this.getLinkedElement()
+      this.$linkedElement = $linkedElement
+    } catch (errorCause) {
+      const cause =
+        errorCause instanceof Error ? errorCause : new Error(String(errorCause))
+      throw new MissingElementError('The linked HTML element does not exist', {
+        cause
+      })
```

Comment:
Mind if we simplify this a bit? TypeScript doesn't know we only throw `Error` but we do

So can either leave the right hand side as `undefined` because it's never used

```suggestion
    } catch (cause) {
      throw new MissingElementError('The linked HTML element does not exist', {
        cause: cause instanceof Error ? cause : undefined
      })
```

Reply:
Feels a shame to add extra code so could ignore `@typescript-eslint/no-unsafe-assignment` too